radis/lbl/factory.py

```diff
@@ -1542,6 +1588,82 @@ def _generate_wavenumber_arrays(self):
 
         return
 
+    def predict_time(self):
+        def _is_at_equilibrium():
```

Comment:
@erwanp I was using the `_is_at_equilibrium` function in `predict_time()` to check whether its non_equilibrium and if optimization = "simple" or "min-RMS" then `x 2` the  calculation time because `_apply_lineshape_DLM` is called twice which is the bottleneck step. One thing I found if we use this function it won't work if using same Spectrum factory because overpopulation gets updated to `{}` if using non_eq_spectrum. Should we modify `self.input.overpopulation = None` when using `eq_spectrum`?